src/util/panic.js

```diff
@@ -36,17 +38,31 @@ function enable_heapdump(name, next_mb, step_mb) {
     return module.exports;
 }
 
+function enable_malloc_print(name, next_mb, step_mb) {
```

Comment:
Perhaps just include it in the regular memory interval instead of adding another?

Reply:
I can, just was thinking we maybe want to set one and not the other, like if I will turn on the heapdump it will kill the process after every 256MB.